docs/products/grafana/howto/pitr-process-for-grafana.rst

```diff
@@ -0,0 +1,29 @@
+Point-in-time recovery (PITR) process for Aiven for Grafana®
+============================================================
+
+When restore from PITR backup for Grafana, a new service will be created.
+
+Steps
+-----
+
+1. Navigate to the backup tab of Grafana service
```

Comment:
Recommend using the new console navigation while describing steps and for screenshots. The old screen (with top nav tabs) will be phased out soon. 

I would rewrite the above line as follows: In the Aiven for Granfana Service, select Backups from the left sidebar.